lib/edit/MonraceDefinitions.jsonc

```diff
@@ -87511,6 +87511,137 @@
         "ja": "斃れた巨人が邪悪な術で再び動き出したものだ。かつてのような力強さは感じられないが、汚れた一撃には注意が必要だ。",
         "en": "A fallen giant, reborn with evil magic. Though it no longer has the strength it once had, its unholy blows must be heeded."
       }
-    }
+    },
+    {
+      "id": 1366,
+      "name": { "ja": "虹色の宝箱", "en": "Legendary Loot box mimic" },
```

Comment:
There's some variation for how names are capitalized, but since this is not a unique, I'd prefer "Legendary loot box mimic".